verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -161,6 +170,9 @@ std::string CreateModuleSignature(absl::string_view);
 // Creates the signature for module instantiations.
 std::string CreateVariableSignature(absl::string_view);
 
+// Creates the signature for funcitons or tasks.
+std::string CreateFunctionOrTaskSignature(absl::string_view);
```

Comment:
Same question, can we separate their signatures?
For instance, functions have return types, and tasks do not.

Reply:
We don't give function return type in Kythe because we aren't interested in it at this time.